Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_Lite_-_Incident_Handling_README.md

```diff
@@ -0,0 +1,75 @@
+- Checks for specific arguments for malicious usage from the command line.
+
+**Verdict:**
+- Determines the appropriate verdict based on the analysis and the investigation findings.
+
+**Verdict Handling:**
+- Handles malicious incidents by initiating appropriate response actions.
+
+---
+
+## Dependencies
+
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+
+* Cortex XDR - Isolate Endpoint
+* Command-Line Analysis
+* Block Indicators - Generic v3
+* Entity Enrichment - Generic v3
+* Cortex XDR - Get entity alerts by MITRE tactics
+
+### Integrations
+
+* CortexXDRIR
+* Cortex XDR - IR
+
+### Scripts
+
+* Set
+* SetAndHandleEmpty
+
+### Commands
+
+* xdr-update-incident
+* xdr-get-incident-extra-data
+
+## Playbook Inputs
+
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| incident_id | Incident ID. | incident.xdrincidentid | Optional |
+| EndpointID | XDR Endpoint ID. | PaloAltoNetworksXDR.Incident.alerts.endpoint_id | Optional |
+| Hostname | Hostname. | PaloAltoNetworksXDR.Incident.alerts.host_name | Optional |
+| Username | User name. | PaloAltoNetworksXDR.Incident.alerts.user_name | Optional |
+| AutoIsolateEndpoint | Whether to isolate the endpoint automatically. | False | Optional |
+| AutoBlockIndicators | Possible values: True/False.  Default: True.<br/>Should the given indicators be automatically blocked, or should the user be given the option to choose?<br/><br/>If set to False - no prompt will appear, and all provided indicators will be blocked automatically.<br/>If set to True - the user will be prompted to select which indicators to block. | False | Optional |
+| UserVerification | Possible values: True/False.  Default: False.<br/>Whether to provide user verification for blocking IPs. <br/><br/>False - No prompt will be displayed to the user.<br/>True - The server will ask the user for blocking verification and will display the blocking list. | False | Optional |
+| XDRRelatedAlertsThreshold | This is the minimum threshold for XDR-related alerts of medium severity or higher, based on MITRE tactics used to identify malicious activity on the endpoint and by the user.<br/>Example: If this input is set to '5' and it detects '6' XDR-related alerts, it will classify this check as indicating malicious activity.<br/>The default value is '5'. | 5 | Optional |
+| InternalRange | This input is used in the "Entity Enrichment - Generic v3" playbook.<br/>A list of internal IP ranges to check IP addresses against. The list should be provided in CIDR notation, separated by commas. An example of a list of ranges is: "172.16.0.0/12,10.0.0.0/8,192.168.0.0/16" \(without quotes\). If a list is not provided, uses the default list provided in the IsIPInRanges script \(the known IPv4 private address ranges\). |  | Optional |
+| XDRDomain | XDR instance domain. | incident.xdrurl | Optional |
```

Comment:
why we need this? or do we need this as a playbook input?

Reply:
I'm using a transformer for this input's value. The playbook then uses this value in a filter to exclude the XDR URL from the entity enrichment sub-playbook. Since I using a transformer initially, I would prefer to use it as an input.